Vue/vaahtwo/stores/settings/store-localization.js

```diff
@@ -467,16 +467,10 @@ export const useLocalizationStore = defineStore({
         getItemMenuList: function (){
             this.item_menu_list =  [
                 {
-                    label: 'Run Seeds',
+                    label: 'Run & Generate Seeds',
```

Comment:
@we-vikram-wri231 this should come from the transaction, in an asset you should return all the strings that we'll use in the specific section. All future developments should use translation by default so that we don't neet to put efforts again.